recipes/neon2sse/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+import os
+from conans import ConanFile, tools, CMake
+
+class neon2sseConan(ConanFile):
+    name = "neon2sse"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/intel/ARM_NEON_2_x86_SSE"
+    description = "Header only library intended to simplify ARM->IA32 porting"
+    license = "BSD-2-Clause"
+    topics = "neon", "sse", "port", "translation", "intrinsics"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
```

Comment:
We could steer this recipe in a completely different direction and remove all the CMake related code (wrapper, build, configure, install, removal of `lib` folder) with a single `self.copy()` since this is a single header file.
I think the current approach is a bit more conservative towards future possible changes (E.g. they decide to add more headers files).